app/components/agenda/agenda-header/agenda-version-actions.js

```diff
@@ -346,6 +359,17 @@ export default class AgendaAgendaHeaderAgendaVersionActions extends Component {
     this.showConfirmForApprovingAgendaAndClosingMeeting = false;
   }
 
+  @action
+  async openConfirmApproveAgendaAndCloseMeetingSidebar() {
```

Comment:
Again with the "sidebar" here... This is really confusing naming.

Reply:
Ok, after testing locally I see where the "sidebar" naming is coming from. It was implemented as a sidebar on usertest, even though it behaves more like a modal. Still, I'd prefer the naming for these properties and functions to be a bit more in line with the actual components it shows/hides.